src/renderer/widgets/VoteModal/ui/formFields/AccountsSelector.tsx

```diff
@@ -34,15 +35,16 @@ export const AccountsSelector = ({ value, accounts, asset, chain, hasError, erro
           id: account.id.toString(),
           value: account,
           element: (
-            <div className="flex w-full justify-between" key={account.id}>
-              <AccountAddress
-                size={20}
-                type="short"
+            <div className="flex w-full items-center justify-between gap-2 text-start text-body" key={account.id}>
+              <Address
                 address={address}
-                name={isShard ? toShortAddress(address, 16) : account.name}
+                variant="truncate"
+                showIcon
+                iconSize={16}
                 canCopy={false}
+                title={isShard ? undefined : account.name}
```

Comment:
why if `isShard` then undefined? Shards also can have names and now we usually try to show that names